### PR TITLE
[trantor] Update to 1.5.1

### PR DESCRIPTION
**Pull request by an-tao**

### CODE REVIEW

File: ports/trantor/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "trantor",
+  "version-semver": "1.5.1",
+  "port-version": 1,
```
Comment (JonLiu1993) [inline ports/trantor/vcpkg.json:4]:
```suggestion
```

File: ports/trantor/vcpkg.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "description": "A non-blocking I/O cross-platform TCP network library, using C++14",
+  "homepage": "https://github.com/an-tao/trantor",
+  "license": "BSD-style license",
+  "dependencies": [
+    "openssl",
+    "c-ares"
+  ],
+  "features": {}
```
Comment (JonLiu1993) [inline ports/trantor/vcpkg.json:12]:
```suggestion
  ]
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/trantor/vcpkg.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "trantor",
+  "version-semver": "1.5.1",
+  "port-version": 1,
+  "description": "A non-blocking I/O cross-platform TCP network library, using C++14",
+  "homepage": "https://github.com/an-tao/trantor",
+  "license": "BSD-style license",
+  "dependencies": [
```
Comment (JonLiu1993) [inline ports/trantor/vcpkg.json:8]: let it depend on vcpkg-cmake port and vcpkg-cmake-config
vcpkg_cmake_configure and vcpkg_cmake_install provided by vcpkg-cmake port.
vcpkg_cmake_config_fixup provide by vcpkg-cmake-config.
eg:  https://github.com/microsoft/vcpkg/blob/master/ports/recast/vcpkg.json

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]
Replace vcpkg_configure_cmake by vcpkg_cmake_configure,vcpkg_install_cmake by vcpkg_cmake_install,vcpkg_fixup_cmake_targets by vcpkg_cmake_config_fixup

File: ports/trantor/vcpkg.json (hunk 2 above)
Comment (JonLiu1993) [inline ports/trantor/vcpkg.json:12]:
```suggestion
    "openssl",
    "c-ares",
    {
      "name": "vcpkg-cmake",
      "host": true
    },
    {
      "name": "vcpkg-cmake-config",
      "host": true
    }
  ]
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by JonLiu1993** [comment]
@an-tao ,Could you please take a look:
https://dev.azure.com/vcpkg/public/_build/results?buildId=57527&view=artifacts&pathAsName=false&type=publishedArtifacts

**Comment by an-tao** [comment]
@JonLiu1993 Thanks so much for your help, but I don't know why it runs the `ninja -v` command. here is the error:

```shell
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:127 (message):
    Command failed: ninja -v
```

**Comment by JonLiu1993** [comment]
@an-tao ,You did not find the real error, this is the core error:
[config-x64-windows-out.log](https://github.com/microsoft/vcpkg/files/6965313/config-x64-windows-out.log)

**Comment by JonLiu1993** [comment]
@an-tao ,Could you please take a look?

**Comment by an-tao** [comment]
> @an-tao ,Could you please take a look?

@JonLiu1993 Sorry for the late reply, I have no idea about the error, It seems a permission error, do you have any suggestions?

**Comment by JonLiu1993** [comment]
> > @an-tao ,Could you please take a look?
> 
> @JonLiu1993 Sorry for the late reply, I have no idea about the error, It seems a permission error, do you have any suggestions?

I will try to reproduce locally, if it can be reproduced, I will solve it

**Comment by an-tao** [comment]
> > > @an-tao ,Could you please take a look?
> > 
> > 
> > @JonLiu1993 Sorry for the late reply, I have no idea about the error, It seems a permission error, do you have any suggestions?
> 
> I will try to reproduce locally, if it can be reproduced, I will solve it

Thank you so much.

**Comment by JonLiu1993** [comment]
The 'paraview' breaks due CI basline error,just wait CI error fixed.

**Review by JonLiu1993 — APPROVED** [review]

**Review by JonLiu1993 — APPROVED** [review]

**Review by BillyONeal — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Pull request merged**